### PR TITLE
Adding blog posts for spring 2024 meeting + summer school

### PR DESCRIPTION
**Pull request by jibarnum**
Take two: checking for glaring errors please and thank you!

### CODE REVIEW

File: _posts/2024-01-19-pyhc-summer-school-2024-announced.markdown [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title: "A Much-anticipated Sequal: the PyHC 2024 Summer School"
```
Comment (sapols) [inline _posts/2024-01-19-pyhc-summer-school-2024-announced.markdown:3]: I wasn't sure about this capitalization so I checked, and in standard title case the "Anticipated" would be capitalized in "Much-Anticipated". Can you please capitalize it?

File: _posts/2024-01-19-spring-2024-meeting-announcement.markdown [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+The meeting's web page can be found [here]({% link _pages/meetings/community_meetings/spring2024.md %}). **Registration—which is free but required—can be found [here](https://forms.gle/WAB9fpiFygdBZo7n6).** The meeting's web page also has a link to registration, as well as the Zoom telecon information, links to important documents, etc. If you'd like to join our mailing list to learn more information about the meeting and other upcoming events, please see the [PyHC Contact page](http://heliopython.org/contact/) for instructions. You'll also find links there for joining our Element and Helionauts chat groups (where we discuss various PyHC issues/topics, as well as discussing other Python-related questions), as well as a link to connect with PyHC on Twitter!
```
Comment (sapols) [inline _posts/2024-01-19-spring-2024-meeting-announcement.markdown:11]: The link can be made `_pages/meetings/spring2024.md` if you just give the page that permalink when you create it. Please update to match the rest of our URLs.

**Review by sapols — CHANGES_REQUESTED** [review]
The blogs posts look good!
My only requested changes are to capitalize that one letter and remove `community_meetings` from that one link.

File: _posts/2024-01-19-pyhc-summer-school-2024-announced.markdown (hunk 1 above)
Comment (jibarnum) [reply _posts/2024-01-19-pyhc-summer-school-2024-announced.markdown:3]: valid; I wondered that myself.

File: _posts/2024-01-19-spring-2024-meeting-announcement.markdown (hunk 2 above)
Comment (jibarnum) [reply _posts/2024-01-19-spring-2024-meeting-announcement.markdown:11]: sounds good. Removed and will put correct link in once spr '23 page is live.

**Review by sapols — APPROVED** [review]

**Pull request merged**